apps/barnepensjon-ui/src/hooks/useCountries.ts

```diff
@@ -22,6 +22,21 @@ interface Country {
     }
 }
 
+export const moveMostUsedCountriesToBeginning = (allCountries: Country[]) => {
+    const countryStrings = ['NORGE', 'DANMARK', 'SVERIGE'].reverse()
+
+    countryStrings.forEach((countryString) => {
+        const country = allCountries.find(country => country.beskrivelser.nb.tekst === countryString)
+        if (country) {
+            const countryIndex = allCountries.findIndex((allCountry) => allCountry === country)
+            allCountries.splice(countryIndex, 1)
+            allCountries.unshift(country)
+        }
+    })
```

Comment:
Hvorfor ikke bare bruke filter?

Reply:
Har ikke tatt i bruk filter for å få landene i samme rekkefølge som man har skrevet inn i `countryStrings`. 

Så det som skjer er da
1. Finn landet basert på navnet
2. Hvis landet finnes, finn index
3. Fjern landet fra listen, for å unngå duplikater
4. Legg til landet på toppen av listen.

Kunne brukt filter, men da vil rekkefølgen bli annerledes